src/util/persistent_logger.js

```diff
@@ -11,10 +11,10 @@ const dbg = require('./debug_module')(__filename);
 
 /**
  * PersistentLogger is a logger that is used to record data onto disk separated by newlines.
- * 
+ *
```

Comment:
please turn off the space changes in comments - this is causing infinit line changes as most other developers do not have this trimming for comment lines.

Reply:
This is due to the `.editorconfig`. I will remove these changed from this PR and will update the `.editorconfig` in another one. Sounds good?

Reply:
yes thanks